### ISSUE TITLE
Fallback error on t([])

### ISSUE THREAD

**Issue by garrettlancaster** [svenfuchs/i18n#104]
Fallbacks are not working when fetching multiple strings.

Steps to reproduce:
1. set up fallback to default language:
   - `require "i18n/backend/fallbacks"`
   - `I18n::Backend::Simple.send(:include, I18n::Backend::Fallbacks)`
2. try to fetch multiple strings that are missing in the current locale, but present in the default locale:
   - `I18n.t([:missing, :missing2])`

**Comment by radar** [comment]
To confirm, the bug here is that when you do:

```
irb(main):004:0> I18n.t([:hello, :world])
```

It doesn't read the translations from the fallback locale, but instead outputs a message like:

```
=> "translation missing: pl.world"
```

Yeah?

**Comment by radar** [comment]
Should be fixed by #304.
